doc/man/8/rbd.rst

```diff
@@ -237,6 +237,8 @@ Commands
 :command:`cp` (*src-image-spec* | *src-snap-spec*) *dest-image-spec*
   Copy the content of a src-image into the newly created dest-image.
   dest-image will have the same size, object size, and image format as src-image.
+  Note that snapshots are not copied. Use `deep cp` command to copy
+  the image including snapshots.
```

Comment:
```suggestion
  Note: snapshots are not copied, use `deep cp` command to include
  snapshots.
```

Reply:
OK, pull request updated according to your suggestions.